thirdeye-ui/src/app/components/home-page/entity-search/entity-search.component.tsx

```diff
@@ -56,8 +58,18 @@ export const EntitySearch: FunctionComponent<EntitySearchProps> = ({
             });
         }
 
+        if (anomalies) {
+            anomalies.forEach((anomaly) => {
+                options.push({
+                    type: t("label.anomaly"),
+                    label: `${anomaly.alert.name} : ${anomaly.id}`,
+                    link: getAnomaliesViewPath(anomaly.id),
+                });
+            });
+        }
```

Comment:
Was this part of the scope for the ticket?

Reply:
Ah, it was just in the designs. Can't find it in the ticket scope.

I'll ask about this in the channel.

![Screenshot_150592396](https://user-images.githubusercontent.com/20799363/206141122-8f381c5a-cb31-421b-a862-62d426ead695.png)

Reply:
Madhumita said it's not required as of now, so I'll remove this

Thread:
https://cortex-data.slack.com/archives/C031NQQNDPX/p1670423198556289?thread_ts=1669844084.782179&cid=C031NQQNDPX